core/src/main/java/org/infinispan/manager/EmbeddedCacheManager.java

```diff
@@ -182,11 +182,15 @@
     */
    boolean cacheExists(String cacheName);
 
+   <K, V> Cache<K, V> getCache(String cacheName, String configurationName);
```

Comment:
Some javadoc would be nice to explain how this differs from the other getCache method.